src/schema.ts

```diff
@@ -273,44 +277,17 @@ export interface FlywheelConnectedEvent {
 }
 
 /**
- * Represents every channel that can be played on the vibraphone.
- *
- * We decided to start at one since the [[BassString]] type also
- * starts at one. We also decided to only represent playable
- * channels and not actual channels (two per note) because that is
- * an implementation detail of the physical MMX.
+ * All possible events relating to the machine itself.
  */
-export type VibraphoneChannel = 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11;
```

Comment:
You have Channel at the top with "vibraphone" as one of the types. How is that Channel and the VibraphoneChannel related?

Reply:
The usage of these is apparent by reviewing the schema I think?  If not could you be more specific